internal/bcdb/worldstate_query_processor.go

```diff
@@ -162,15 +161,27 @@ func (q *worldstateQueryProcessor) executeJSONQuery(dbName, querierUserID string
 		}
 	}
 
-	keys, err := q.jsonQueryExecutor.ExecuteQuery(dbName, query)
+	snapshots, err := q.db.GetSnapshot(
```

Comment:
nit - why result of `GetSnapshot` assign to variable `snapshots`

Reply:
Done. I have changed the signature to `GetDBsSnapshot` and `DBsSnapshot` in leveldb package.